modules/ROOT/pages/monitoring/notifications.adoc

```diff
@@ -75,6 +75,13 @@ image::notifications-webhook-add.png[width=800]
 
 image::notifications-webhook-remove.png[width=800]
 
+.Accessibility of webhook URL
```

Comment:
"Accessibility" is probably not the word here, as it makes me think of a11y. Perhaps something like "Using webhooks behind a proxy" or "When behind a proxy"? We could also consider having a short bit about configuring proxy settings in the server setup section as it will now also affect the update checker, and then link to that section from here.

Reply:
Its probably not yet perfect :smile: but please have a look at [51e4df3](https://github.com/neo4j/docs-ops-manager/pull/200/commits/51e4df3f74358a7cc244c9f547e51e2de186a61d)